packages/genesys-spark-components/src/components/stable/gux-time-picker/gux-time-picker.tsx

```diff
@@ -109,11 +119,27 @@ export class GuxTimePicker {
     this.i18n = await buildI18nForComponent(this.root, translationResources);
     this.clockType = this.clockType || getLocaleClockType(this.root);
 
+    this.validateValueFormat();
     if (this.clockType == '12h' && (this.min || this.max)) {
-      console.error('clock type must be "24h" when using min/max props');
+      logError(this.root, 'clock type must be "24h" when using min/max props');
+    }
+  }
+
+  @Watch('value')
+  private validateValueFormat() {
+    if (!this.isValidTimeFormat()) {
+      logError(
+        this.root,
+        `"${this.value}" is not a valid value format. Format must be "hh:mm" or "h:mm". Falling back to 00:00 default value`
+      );
+      this.value = '00:00';
     }
```

Comment:
This might be a little nicer if we use the new and old values passed into the @Watch function
```
private validateValueFormat(newValue: GuxISOHourMinute, oldValue: GuxISOHourMinute) {
  if (!this.isValidTimeFormat(newValue)) {
    logError(
      this.root,
      `"${newValue}" is not a valid value format. Format must be "hh:mm" or "h:mm".`
    );
    this.value = oldValue;
  }
}

private isValidTimeFormat(value: GuxISOHourMinute) {
  return typeof value === 'string' && /^\d{1,2}:\d{2}$/.test(value);
}
```